lib/modules/showImages.js

```diff
@@ -1773,27 +1784,67 @@ function videoAdvanced(options) {
 		}
 	});
 
-	element.collapse = () => {
+	let playWhenVisibleInterval;
+
+	function playWhenVisible() {
+		const visibleThreshold = 0.33;
+		const isWithinPlayThreshold = () => getPercentageVisibleYAxis(vid) > visibleThreshold;
+
+		const tryPlay = () => {
+			if (!autoplay || (isWithinPlayThreshold() !== vid.paused)) return;
+			else if (vid.paused) vid.play();
+			else vid.pause();
+		};
+
+		playWhenVisibleInterval = setInterval(tryPlay, 150);
```

Comment:
I think `_.throttle(tryPlay, 150, { leading: false })` in a scroll listener would be preferred over `setInterval` so it's not checking unnecessarily, unless this was chosen for a reason (I haven't read through everything yet)